scoper.inc.php

```diff
@@ -110,10 +120,24 @@ function( $file_path, $prefix, $contents ) {
 				$contents = str_replace( "'Google_", "'" . $prefix . '\Google_', $contents );
 				$contents = str_replace( '"Google_', '"' . $prefix . '\Google_', $contents );
 			}
+
+			if ( preg_match( '#.*trustedlogin/client/.*#', $file_path ) ) {
```

Comment:
These rules were enough to get Trusted Login working in the simple scenarios that I have tried, however I've not exercised all of the various scenarios/edge cases etc and there are likely more patches that need to be applied.